app/_hub/reedelk/reedelk-transformer/index.md

```diff
@@ -0,0 +1,205 @@
+---
+name: Reedelk Transformer
+
+publisher: Reedelk
+
+categories:
+  - transformations
+
+type:
+  plugin        
+
+desc: Kong plugin to transform Reedelk requests and responses
+
+description: |
+  The Reedelk  transformer plugin allows transforming the upstream request body or
+  downstream response body by invoking a Reedelk REST flow before hitting the
+  upstream server, or before sending the downstream response back to the client.
+  The plugin allows applying upstream and downstream transformations together as well.
+
+support_url: https://github.com/reedelk/kong-plugin-reedelk-transformer/issues
+  # (Optional) A specific URL of your own for this extension.
+  # Defaults to the url setting in your publisher profile.
+
+source_url: https://github.com/reedelk/kong-plugin-reedelk-transformer
+
+license_type: Apache-2.0
+  # (Optional) For open source, use the abbreviations in parentheses at:
+  # https://opensource.org/licenses/alphabetical
+
+license_url: https://github.com/reedelk/kong-plugin-reedelk-transformer/blob/master/LICENSE.txt
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.1.x
+      - 1.5.x
```

Comment:
The last two releases of community were 2.1.x and 2.0.x. I assume either one of these entries is incorrect and this plugin is supported in either 2.1 and 2.0, or 2.0 and 1.5; or, 2.0.x is simply missing.

Reply:
> The last two releases of community were 2.1.x and 2.0.x. I assume either one of these entries is incorrect and this plugin is supported in either 2.1 and 2.0, or 2.0 and 1.5; or, 2.0.x is simply missing.

Yes, at first I had this as Kong Enterprise compatible so the numbers are wrong. Will fix and add KE back after they verify.

Reply:
![image](https://user-images.githubusercontent.com/3756245/88982809-dfca7d80-d28e-11ea-9e5b-ce5756d62086.png)

@reedelk please confirm compatible community versions, and enterprise editions versions when you get your trial version tested.

# enterprise_edition:
  #  compatible:
  #    - 2.1.x
  #    - 1.5.x

Reply:
@tlblessing here is the compatibility matrix for the Community Edition and Enterprise Edition:

#### Community Edition
* 2.1.x
* 2.0.x
* 1.5.x

Deployment Strategy:
* "Classic": Full Support
* "DB-less": Full Support
* "Hybrid mode": Full Support

#### Enterprise Edition

* 1.5.x

Deployment Strategy:
* "Classic": Full Support
* "DB-less": Full Support
* "Hybrid mode": Full Support

Thank you

Reply:
@reedelk Thank you for your responses, Lorenzo. Please note that the deployment modes are a bit different for Kong Enterprise:

![image](https://user-images.githubusercontent.com/3756245/89038920-d1b84380-d306-11ea-898c-b08a72e50571.png)

Can you let us know about your support for K4K8s?

Reply:
@tlblessing thanks for pointing that out. I confirm that the plugin is compatible with the Kong Ingress Controller (K4K8s) as well.